gui/components/swipe/index.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect } from "react";
+import { SafeAreaView, StyleSheet, View, Text, Dimensions } from "react-native";
+import SwipeableCard from "../../components/swipe/SwipeableCard";
+import SwipeCount from "./swipeCounts";
+const SCREEN_WIDTH = Dimensions.get("window").width;
+
+const Swipe = ({ cards }) => {
```

Comment:
Can you update the file name to something more specific than index? Is this a container that pulls in Swipeable cards? If this is a page definition, should it go under (tabs)?